docs/setup.md

```diff
@@ -212,19 +212,22 @@ To build Linux, Mac, and Windows wheels using GitHub Actions, create a `.github/
             os: [ubuntu-20.04, windows-2019, macos-10.15]
 
         steps:
-          - uses: actions/checkout@v2
+          - uses: actions/checkout@v3
 
           # Used to host cibuildwheel
-          - uses: actions/setup-python@v2
+          - uses: actions/setup-python@v4
+            with:
+              python-version: 3.x
 
           - name: Install cibuildwheel
             run: python -m pip install cibuildwheel==2.6.1
 
           - name: Build wheels
             run: python -m cibuildwheel --output-dir wheelhouse
 
-          - uses: actions/upload-artifact@v2
+          - uses: actions/upload-artifact@v3
             with:
+              name: wheels
```

Comment:
I'm not sure it's really needed, I usually leave this in `artifacts`, and download them from there. Especially since I'm also uploading the SDist to `artifacts` too; it's not an SDist.

Reply:
The default is `artifact`, not `artifacts`. ) I had to look up the docs to get where the default is coming from.

https://github.com/pypa/cibuildwheel/blob/cd7ea913e6275034ab813c9f73af6c572522d914/examples/github-deploy.yml#L51-L55

I though to remove the `artifact`, but then the [docs](https://github.com/actions/download-artifact#download-all-artifacts) said that if name is not provided, there will be an extra dir, so files in `dist` will end up being in `dist/artifact`. Need to check if this is true.

Reply:
Oops, fixed above. You do need to specify the name (`artifact`) when you use download artifact, or it will inject the artifact name into the extraction directory (since it supports multiple artifact names). That's still less configuration than adding a name on every upload-artifacts, though. Maybe a little note in the text stating that `artifact` is the default artifact name?

Reply:
This implicit unarchive logic hits me in the foot once in a while. Filled how I'd like to see it work https://github.com/actions/download-artifact/issues/166 and will leave it as is for now.

Reply:
Removed the attempt to have explicit name for the artifact.